crates/criticalup-cli/src/commands/install.rs

```diff
@@ -111,6 +111,12 @@ async fn install_product_afresh(
         .await?;
     let verified_release_manifest = release_manifest_from_server.signed.into_verified(&keys)?;
 
+    if release.starts_with('@') {
```

Comment:
We have a couple places we check for this `@` symbol, I wonder if we should teach the `ProjectManifestProduct` how to know if it's an alias or not to isolate concept. Something like a `fn release_alias(&self) -> Option<String>` which returns the alias if any. (And `release()` maybe needs to be rethought too, as if it's an alias it's not actually a release, is it?)